main.tf

```diff
@@ -0,0 +1,12 @@
+
+resource "null_resource" "ansible_provisioner" {
+
+  provisioner "local-exec" {
+    command = "ansible-playbook ${var.playbook} -e host=${var.ip}"
```

Comment:
Also, lets standardize variable names to be similar to ansible arguments. In this case `var.ip` should be `var.host`